etc/openqa/openqa.ini

```diff
@@ -117,7 +117,8 @@ blacklist = job_grab job_done
 
 # Default limits for cleanup (sizes are in GiB, durations in days, zero denotes infinity)
 [default_group_limits]
-#asset_size_limit = 100
+#asset_size_limit = 100 # enforced on job group level for job groups without parent
+#parent_asset_size_limit = 500 # enforced on parent group level
```

Comment:
Do we need two different default levels? I think we should be ok to have just the default as in before

Reply:
I agree. For migration I would set the job group parent's limit to the sum of the job group's limits  though.